cmd/collection/main.go

```diff
@@ -396,8 +401,40 @@ func main() {
 
 			signer := verification.NewSingleSigner(staking, node.Me.NodeID())
 
+			// create flow client with correct GRPC configuration for QC contract client
+			var flowClient *client.Client
+			if insecureAccessAPI {
+				flowClient, err = common.InsecureFlowClient(accessAddress)
+				if err != nil {
+					return nil, err
+				}
+			} else {
+				if secureAccessNodeID == "" {
+					return nil, fmt.Errorf("invalid flag --secure-access-node-id required")
```

Comment:
```suggestion
					return nil, fmt.Errorf("invalid flag, --secure-access-node-id required")
```
otherwise the reader may think the invalid qualifier applies to your requirement